ddtrace/appsec/iast/_taint_utils.py

```diff
@@ -30,19 +36,27 @@ def get(self, key, default=None):
 
     def items(self):
         for k, v in super(LazyTaintDict, self).items():
+            if k and isinstance(k, (str, bytes, bytearray)) and not is_pyobject_tainted(k):
+                try:
+                    k = taint_pyobject(k, Input_info(k, k, self.origin_key))
+                except Exception:
+                    log.debug("Unexpected exception while tainting key", exc_info=True)
```

Comment:
Should we log the key/value pairs. This could help with debuggability. Would this be a security risk?

Reply:
We could leak sensitive data as we would be logging http request headers, it's likely that authentication information could be there (i.e. JWT tokens)